python/lsst/ts/mtdomecom/mock_llc/apscs.py

```diff
@@ -150,6 +150,16 @@ async def _warn_invalid_state(self, shutter_id: int) -> None:
     async def _handle_open_or_close_or_stationary(
         self, current_tai: float, shutter_id: int
     ) -> None:
+        # TODO DM-50801 Remove the mappings below this line as soon as the enum
+        #  values are available in ts_xml.
+        try:
+            floce = MotionState.FINAL_DOWN_CLOSE_LS_ENGAGED
+        except AttributeError:
+            floce = MotionState.FINAL_LOW_CLOSE_LS_ENGAGED
+        try:
+            flole = MotionState.FINAL_DOWN_OPEN_LS_ENGAGED
+        except AttributeError:
+            flole = MotionState.FINAL_LOW_OPEN_LS_ENGAGED
```

Comment:
As the discussion, please help to check the variable names of `floce` and `flole`.

Reply:
Done